plugins/mcreator-localization/lang/texts.properties

```diff
@@ -55,11 +55,41 @@ action.gradle.close_mcreator_while_installation_message=<html><b>Gradle is still
+dialog.ai_condition.additional_continue=Additional continue condition
+dialog.ai_condition.no_additional=(no additional condition)
+dialog.ai_condition.panel_name=AI condition editor
+dialog.animation_editor.title_wizard=Model animation definition wizard
+dialog.animation_editor.action_set_new=Set new animations
+dialog.animation_editor.action_keep_current=Keep current animations
+dialog.animation_editor.action_set=Set animations
 dialog.common.save_changes=Save changes
 dialog.cache_cleanup.title=Gradle caches cleanup
 dialog.cache_cleanup.progress.stopping_daemons=Stopping running Gradle daemons
 dialog.cache_cleanup.progress.clearing_gradle_caches_folder=Clearing the Gradle caches folder
 dialog.cache_cleanup.progress.build_task=Running Gradle build task
+dialog.code_error.compilation_desc=<html><b color=#ffc40d>Your main mod file did not compile properly.</b><br><br>This is usually caused by inconsistencies in the mod code during builds.<br>MCreator can solve this issue for you by regenerating entire mod's code.<br><br>If you have locked mod elements or mod base files, they can cause this error too.</small><br><br>Click <b>Regenerate code</b> button to proceed with this action.<br>
+dialog.code_error.compilation_title=Main mod file compilation errors
+dialog.code_error.compilation_list=<html><b color=#ffc40d>Your workspace contains mod elements that don't compile properly.</b><br><br>This was most likely caused by the last mod element you added or edited. Try altering the settings of mod elements<br>that cause build errors (such elements are listed below) to try to fix this problem.<br><br>If you can't resolve this problem, export the workspace to shareable ZIP file from the <i>File</i> menu and send it to us via<br>our Issue tracker on our website so we can try to help you.<br><br><b>List of mod elements that cause compilation errors (trimmed to up to 10 elements):<br><br>
+dialog.element_order.instructions=Click on the element or selection and drag to reorder them
+dialog.element_order.editor_title=Creative tab element order editor
+dialog.element_order.change_message=<html>MCreator will now rebuild the workspace code to apply changes.<br><br><b>Existing map saves will keep the old order!
+dialog.element_order.change_title=Elements order change
+dialog.file.error_save_inside_workspace_root_message=You can not make a workspace inside workspaces root folder. Make a subdirectory here first!
+dialog.file.error_save_inside_workspace_root_title=Workspace error
+dialog.file.error_save_inside_folder_not_empty_message=The directory you have selected for the workspace is not empty!
+dialog.file.error_save_inside_folder_not_empty_title=Directory not empty
+dialog.file.error_invalid_path=<html>The directory you have selected for the workspace has invalid path!<br><br>You selected the following directory:<br><kbd> {0}</kbd><br><br>Only English letters, numbers, <kbd>_/+-\\:()[].,@$=`</kbd> and whitespace are allowed in the path.
```

Comment:
Did you verify if this one is printed properly?, some of the chars in the list might need escaping per https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#load(java.io.Reader)

Reply:
I verified now and it is properly escaped, at least per IDEAs checks.